src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static final int number = 4;
+
+    public static void main(String[] args) {
+        Figure[] array = new Figure[number];
+        Random random = new Random();
+        int numberDifferentFigure = 5;

```

Comment:
Avoid using magic numbers directly in your code. Consider defining a constant for the number of different figures, as suggested in the checklist: `public static final int FIGURE_COUNT = 5;` and use it in the `getRandomFigure` method.